tests/e2e/pageobjects/ide/Ide.ts

```diff
@@ -219,31 +270,30 @@ export class Ide {
     }
 
     async closeAllNotifications() {
-        const notificationLocator: By = By.css('.theia-Notification');
-
+        const statusBarLocator: By = By.id('theia-statusBar');
+        const statusBarNotificationsLocator: By = By.xpath('.//div[@class="area right"]/div[@class="element hasCommand"][contains(@title, "Notification")]');
+        const notificationCenterLocator: By = By.xpath('//div[@class="theia-notifications-container theia-notification-center open"]');
+        const notificationCenterActionsLocator: By = By.xpath('.//div[@class="theia-notification-center-header"]/div[@class="theia-notification-center-header-actions"]/ul[@class="theia-notification-actions"]');
+        const notificationCenterActionCloseAllButtonLocator: By = By.xpath('.//li[@class="clear-all"]');
         Logger.debug('Ide.closeAllNotifications');
 
-        if (! await this.driverHelper.isVisible(notificationLocator)) {
-            return;
-        }
-
-        const notifications: WebElement[] = await this.driverHelper.waitAllPresence(notificationLocator);
-        const notificationsCapacity: number = notifications.length;
-
-        for (let i = 1; i <= notificationsCapacity; i++) {
-            const notificationLocator: By = By.xpath('//div[@class=\'theia-Notification\']//button[text()=\'Close\']');
-
-            try {
-                await this.driverHelper.waitAndClick(notificationLocator);
-            } catch (err) {
-                if (err instanceof error.TimeoutError) {
-                    console.log(`The '${notificationLocator}' element is not visible and can't be clicked`);
-                    continue;
-                }
-
-                throw err;
-            }
-        }
+        Logger.debug('Ide.closeAllNotifications getting statusBar element');
+        const statusBar: WebElementPromise = (await this.driverHelper.getDriver()).findElement(statusBarLocator);
+        Logger.debug('Ide.closeAllNotifications getting statusBar notifications button');
+        const notificationsButton: WebElementPromise = (await statusBar).findElement(statusBarNotificationsLocator);
+        Logger.debug('Ide.closeAllNotifications clicking notifications button');
+        await (await notificationsButton).click();
+
+        Logger.debug('Ide.closeAllNotifications waiting for Notifications center to become visible [5s].');
+        await this.driverHelper.waitVisibility(notificationCenterLocator, 5_000);
```

Comment:
This line seems redundant to me as the code is waiting for that element on line 290.

Reply:
not sure if still applies